src/stores/recordsStore.ts

```
@@ -92,34 +88,45 @@ export const useRecordsStore = defineStore("recordsStore", () => {
-      pt3: getRandomScore(70, 100),
-      pt4: getRandomScore(50, 100),
-      pt5: getRandomScore(70, 100),
-      pt6: getRandomScore(20, 100),
-      pt7: getRandomScore(70, 100),
-      pt8: getRandomScore(70, 100),
-      pt9: getRandomScore(50, 100),
-      pt10: getRandomScore(50, 100),
-      qa1: getRandomScore(50, 100),
-      class_record_id: classRecordId,
-    }));
+    const records: Record[] = students.map((student: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)